backend/app/src/main/java/com/moh/phlat/backend/repository/ProcessDataRepository.java

```diff
@@ -14,19 +16,139 @@ public interface ProcessDataRepository extends JpaRepository<ProcessData, Long>
+	@Query(value="Select distinct pd.hdsName FROM ProcessData pd WHERE pd.controlTableId = :controlTableId")
+	public List<String> findAllDistinctHdsName(Long controlTableId);
+	//@Query(value="Select distinct pd.hds_name_alias FROM ProcessData pd WHERE pd.controlTableId = :controlTableId")
+	//public List<String> findAllDistinctHdsNameAlias(Long controlTableId);
+	@Query(value="Select distinct pd.hdsPreferredNameFlag FROM ProcessData pd WHERE pd.controlTableId = :controlTableId")
+	public List<String> findAllDistinctHdsPreferredNameFlag(Long controlTableId);
+	@Query(value="Select distinct pd.hdsEmail FROM ProcessData pd WHERE pd.controlTableId = :controlTableId")
+	public List<String> findAllDistinctHdsEmail(Long controlTableId);
+	@Query(value="Select distinct pd.hdsWebsite FROM ProcessData pd WHERE pd.controlTableId = :controlTableId")
+	public List<String> findAllDistinctHdsWebsite(Long controlTableId);
+	@Query(value="Select distinct pd.hdsBusTelAreaCode FROM ProcessData pd WHERE pd.controlTableId = :controlTableId")
+	public List<String> findAllDistinctHdsBusTelAreaCode(Long controlTableId);
+	@Query(value="Select distinct pd.hdsBusTelNumber FROM ProcessData pd WHERE pd.controlTableId = :controlTableId")
+	public List<String> findAllDistinctHdsBusTelNumber(Long controlTableId);
+	@Query(value="Select distinct pd.hdsTelExtension FROM ProcessData pd WHERE pd.controlTableId = :controlTableId")
+	public List<String> findAllDistinctHdsTelExtension(Long controlTableId);
+	@Query(value="Select distinct pd.hdsCellAreaCode FROM ProcessData pd WHERE pd.controlTableId = :controlTableId")
+	public List<String> findAllDistinctHdsCellAreaCode(Long controlTableId);
+	@Query(value="Select distinct pd.hdsCellNumber FROM ProcessData pd WHERE pd.controlTableId = :controlTableId")
+	public List<String> findAllDistinctHdsCellNumber(Long controlTableId);
+	@Query(value="Select distinct pd.hdsFaxAreaCode FROM ProcessData pd WHERE pd.controlTableId = :controlTableId")
+	public List<String> findAllDistinctHdsFaxAreaCode(Long controlTableId);
+	@Query(value="Select distinct pd.hdsFaxNumber FROM ProcessData pd WHERE pd.controlTableId = :controlTableId")
+	public List<String> findAllDistinctHdsFaxNumber(Long controlTableId);
+	@Query(value="Select distinct pd.pcnServiceDeliveryType FROM ProcessData pd WHERE pd.controlTableId = :controlTableId")
+	public List<String> findAllDistinctHdsServiceDeliveryType(Long controlTableId);
+	@Query(value="Select distinct pd.pcnClinicType FROM ProcessData pd WHERE pd.controlTableId = :controlTableId")
+	public List<String> findAllDistinctPcnClinicType(Long controlTableId);
+	@Query(value="Select distinct pd.pcnPciFlag FROM ProcessData pd WHERE pd.controlTableId = :controlTableId")
+	public List<String> findAllDistinctPcnPciFlag(Long controlTableId);
+	/*@Query(value="Select distinct pd.HDS_HOURS_OF_OPERATION FROM ProcessData pd WHERE pd.controlTableId = :controlTableId")
```

Comment:
The commented code has been removed.  I wasn't sure what was matching up on the columns on the source and process web pages vs the database.  I was leaving them in case I missed one of the columns on the page.